t3f/initializers.py

```diff
@@ -5,13 +5,200 @@
 from t3f.tensor_train_batch import TensorTrainBatch
 
 
-def random_tensor(shape, tt_rank=2):
-  """Generate a random TT-tensor of given shape.
+def tensor_ones(shape):
```

Comment:
What about batches? Do we want to support them as batch_size=None argument, or create a separate function, or ignore them?